### ISSUE TITLE
[FastestDet] height and width axes are switched before last concatenation

### ISSUE THREAD

**Issue by Hyunseok-Kim0** [PINTO0309/onnx2tf#180]
### Issue Type

Others

### onnx2tf version number

1.6.2

### onnx version number

1.12.0

### tensorflow version number

2.10

### Download URL for ONNX

https://github.com/PINTO0309/onnx2tf/releases/download/1.1.28/fastestdet.onnx

### Parameter Replacement JSON

```json
N/A
```

### Description

1. Purpose: Personal development
2. What: In issue #124, ONNX-Tensorflow validation check of FastestDet gives `Unmatched` for last layer as shown below. 
![Screenshot from 2023-02-09 14-58-08](https://user-images.githubusercontent.com/34959032/217731209-a7503bdb-2618-4df2-bc30-8f65872b7286.png) I found reason why it gives wrong value for last `Concatenation`.

ONNX | TF
---- | ----
<img src="https://user-images.githubusercontent.com/34959032/217731925-bae71fee-257a-47c9-a245-a9b7b555f66c.png" width="150">|  <img src="https://user-images.githubusercontent.com/34959032/217731953-09e75533-ef33-476e-8da1-e9e28a6140c4.png" width="150">

Before last `Concatenation`, height and width axes of last branch are switched. Due to this problem, final tensor between onnx and tensorflow have difference as below.
<img src="https://user-images.githubusercontent.com/34959032/217732634-c97341f7-779e-499f-802f-de1eda984ff8.png" height="300">

**Comment by PINTO0309** [comment]
Thanks for pointing out this issue. I noticed that the special transpositions before and after `Softmax` were not addressed enough. Therefore, I have just released a corrected package.

- After
  ![image](https://user-images.githubusercontent.com/33194443/217760618-35809f57-80e2-485e-8fb4-5612c2769306.png)
  ONNX | TF
  :-: | :-:
  <img src="https://user-images.githubusercontent.com/34959032/217731925-bae71fee-257a-47c9-a245-a9b7b555f66c.png" width="150">|  <img src="https://user-images.githubusercontent.com/33194443/217760365-b1f000fb-34ab-493b-9acc-1a6e406abbf8.png">
